File: Examples/CSharp/AsposePDF/Forms/DeleteFormField.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Forms
{
    public class DeleteFormField
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Forms();

            //open document
            Document pdfDocument = new Document(dataDir + "DeleteFormField.pdf");

            //delete a particular field by name
            pdfDocument.Form.Delete("textbox1");
            
            //save modified document
            pdfDocument.Save(dataDir + "DeleteFormField_out.pdf");

        }
    }
}

File: Examples/VisualBasic/AsposePDF/Forms/DeleteFormField.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.Forms
    Public Class DeleteFormField
        Public Shared Sub Run()
            ' ExStart:DeleteFormField
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Forms()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("DeleteFormField.pdf"))

            ' Delete a particular field by name
            pdfDocument.Form.Delete("textbox1")
            dataDir = dataDir & Convert.ToString("DeleteFormField_out_.pdf")
            ' Save modified document
            pdfDocument.Save(dataDir)
            ' ExEnd:DeleteFormField
            Console.WriteLine(Convert.ToString(vbLf & "Particular field deleted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

